lib/game.rb

```diff
@@ -0,0 +1,89 @@
+class Game
+  include Constable
+
+  attr_accessor :board, :player_1, :player_2
+
+  def initialize(player_1 = Players::Human.new("X"), player_2 = Players::Human.new("O"), board = Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+
+  def current_player
+    return player_1 if board.turn_count.even?
+    player_2
+  end
+
+  def over?
+    won? || draw?
+  end
+
+  def won?
+    b = board.cells
+    WIN_COMBINATIONS.detect do |c|
+      b[c[0]] == b[c[1]] && b[c[1]] == b[c[2]] && b[c[0]] != " "
+    end
+  end
+
+  def draw?
+    board.full? && !won?
+  end
+
+  def winner
```

Comment:
Instead of calling `Game#won?` twice, consider using an if statement assignment. For example:

```ruby
if won = won?
  board.cells[won.first]
end
```

If `won?` returns true, then and only then will the local variable `won` be assigned, otherwise, the method will just abort with a false return value. But if `won` is assigned, we can use that variable instead of calling `won?` again within the if statement.